core/src/main/java/org/jdbi/v3/core/mapper/reflect/FieldMapper.java

```diff
@@ -143,7 +144,7 @@ private Optional<RowMapper<T>> createSpecializedRowMapper(StatementContext ctx,
         for (Class<?> aType = type; aType != null; aType = aType.getSuperclass()) {
             for (Field field : aType.getDeclaredFields()) {
                 Nested nested = field.getAnnotation(Nested.class);
-                if (!JdbiAnnotations.isMapped(field)) {
+                if (Modifier.isStatic(field.getModifiers()) || !JdbiAnnotations.isMapped(field)) {
```

Comment:
are there any other modifiers that we should think of?

Reply:
I don't think so? I suppose you could argue `transient` but I don't want to potentially break user code without at least some justification. Are you thinking of any specific modifier?

Reply:
We can always add more exclusions in future changes.